src/python/install.sh

```diff
@@ -780,6 +780,12 @@ if [ "${INSTALL_JUPYTERLAB}" = "true" ]; then
     install_user_package $INSTALL_UNDER_ROOT jupyterlab
     install_user_package $INSTALL_UNDER_ROOT jupyterlab-git
 
+    if [ "$INSTALL_UNDER_ROOT" = false ]; then
```

Comment:
Can you bump the patch version of the Feature? Thanks! 

https://github.com/devcontainers/features/blob/4ccebdd6bc93110c4e7c1c8a5cb2f3c9df706cc7/src/python/devcontainer-feature.json#L3